lc-hax/Scripts/Commands/EntranceCommand.cs

```diff
@@ -4,6 +4,12 @@
 namespace Hax;
 
 public class EntranceCommand : ICommand {
+    public bool forceInside = false;
+
+    public EntranceCommand(bool inside = false) {
+        this.forceInside = inside;
+    }
```

Comment:
Let's not use constructors for commands. Eventually, I want to migrate to populating the `Commands` dictionary using attributes and reflection, and this would be a blocker then. It's alright to create two different scripts and then abstract a common method.

Reply:
can I suggest accepting it as-is and refactoring it later as needed?

Reply:
I can refactor for you, it's not a problem.